Xamarin.ExposureNotification/ExposureInfo.shared.cs

```diff
@@ -32,16 +33,28 @@ public ExposureInfo(DateTime timestamp, TimeSpan duration, int attenuationValue,
 
 	public class ExposureDetectionSummary
 	{
+		[EditorBrowsable(EditorBrowsableState.Never)]
 		public ExposureDetectionSummary(int daysSinceLastExposure, ulong matchedKeyCount, byte maximumRiskScore)
-			: this(daysSinceLastExposure, matchedKeyCount, maximumRiskScore, null, 0)
+			: this(daysSinceLastExposure, matchedKeyCount, (int)maximumRiskScore, null, 0)
 		{
 		}
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
 		public ExposureDetectionSummary(int daysSinceLastExposure, ulong matchedKeyCount, byte maximumRiskScore, TimeSpan[] attenuationDurations, int summationRiskScore)
+			: this(daysSinceLastExposure, matchedKeyCount, (int)maximumRiskScore, attenuationDurations, summationRiskScore)
+		{
+		}
+
+		public ExposureDetectionSummary(int daysSinceLastExposure, ulong matchedKeyCount, int highestRiskScore)
+			: this(daysSinceLastExposure, matchedKeyCount, highestRiskScore, null, 0)
+		{
+		}
+
+		public ExposureDetectionSummary(int daysSinceLastExposure, ulong matchedKeyCount, int highestRiskScore, TimeSpan[] attenuationDurations, int summationRiskScore)
 		{
 			DaysSinceLastExposure = daysSinceLastExposure;
 			MatchedKeyCount = matchedKeyCount;
-			MaximumRiskScore = maximumRiskScore;
+			HighestRiskScore = highestRiskScore;
```

Comment:
Not only added a new prop, but also changed the arg name so we can pretend this never happened.